### DISCUSSION TITLE
是否支持自定义的登录系统

### DISCUSSION THREAD

**Discussion by elainecyx** [walinejs/waline#1909] [Q&A]
登录不用社交账号，期望能接入到自己项目的登录系统，是否可以做到呢？感谢帮助！

**Comment by lizheming** [comment]
这个有在计划中，暂时还木有进度，可以一块提一些方案~

**Reply by dengji85** [reply]
比如可以支持自定义配置oauh2参数，那使用自己单位的oauh2登陆系统就可以直接对接了

**Reply by lizheming** [reply]
这个本身就是支持的，自定义 OAUTH_URL 就可以了

**Comment by lizheming** [comment]
https://waline.js.org/cookbook/customize/userdb.html 可以看下这个文档。
